frontend/views/recipes-list-view.ts

```diff
@@ -123,6 +123,7 @@ export class RecipesListView extends LitElement {
           display: block;
           --recipes-filter-column-width: 200px;
           --recipes-list-view-header-height: 80px;
+          background-color: var(--color-alloy-lighter);
```

Comment:
I think this should be applied to the html element, not just for the recipes-list-view, so that it applies in the individual recipe view as well.

Reply:
Good enough for now. Should perhaps revisit eventually.